action/secret/file.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package secret
+
+import "github.com/go-vela/types/library"
+
+// ConfigFile represents the secret configuration necessary
+// to perform secret related quests from a file with Vela.
```

Comment:
```suggestion
// to perform secret related requests from a file with Vela.
```
though "quests" sounds cooler 🗡️ 🐉

Reply:
@wass3r yeah what can I say? I was trying to inspire folks with my description 🤪 

But in all seriousness, good catch 👍